src/domain/reservation/manage/action/ManageReservationsDropdown.js

```diff
@@ -22,6 +25,8 @@ const ManageReservationsDropdown = ({ t }) => {
 };
 
 ManageReservationsDropdown.propTypes = {
-  t: PropTypes.func
+  t: PropTypes.func,
+  toggleModal: PropTypes.func,
```

Comment:
I don't thin `toggleModal` is a good callback function name. This component doesn't know what does the callback do. It should be named into something like `onInfoClick`.